src/Sylius/Bundle/AdminBundle/Resources/views/Customer/Show/_content.html.twig

```diff
@@ -7,30 +7,37 @@
                 </a>
                 <div class="meta">
                     <span class="date">{{ 'sylius.ui.customer_since'|trans }} {{ customer.createdAt|date }}</span>
+                    <br />
+                   {% if customer.group %}
```

Comment:
But it should be `!== null`, because when variable is null then it's already defined, WDYT?